docs/turborepo.md

```diff
@@ -0,0 +1,38 @@
+# Turborepo and Telescope
+
+Telescope is configured as a monorepo: a single repository that holds many projects. [Turborepo](https://turborepo.org/) is the tool we have chosen to manage every project within Telescope.
```

Comment:
Instead of we have chosen, probably could say we use

"Turborepo is the tool we use to mange every project within Telescope"